lm_eval/tasks/aexams/README.md

```diff
@@ -0,0 +1,49 @@
+# Arabic EXAMS
+
+### Paper
+
+EXAMS: a resource specialized in multilingual high school exam questions.
+The original paper [EXAMS](https://aclanthology.org/2020.emnlp-main.438/)
+
+The Arabic EXAMS dataset includes 5 subjects 
+     
+  - Islamic studies
+  - Biology 
+  - Physics 
+  - Science 
+  - Social
+
+The original dataset [EXAMS-QA](https://github.com/mhardalov/exams-qa)
+
+EXAMS is a benchmark dataset for cross-lingual and multilingual question answering for high school examinations.
+With 24,000 high-quality high school exam questions in 16 languages, covering 8 language families and 24 school subjects from Natural Sciences and Social Sciences, among others.
+EXAMS offers unique fine-grained evaluation framework across multiple languages and subjects
+
+Homepage for Arabic EXAMS: [EXAMS Arabic Homepage](https://github.com/FreedomIntelligence/AceGPT/tree/main/eval/benchmark_eval/benchmarks/EXAMS_Arabic)
+
+### Citation
+
+
+### Groups and Tasks
+
+#### Groups
+
+- `EXAMS Arabic`: include IslamicStudies, Biology, Science, Physics, Social.
+
+#### Tasks
+
+
+The following tasks evaluate subjects in Arabic EXAMS dataset using loglikelihood-based multiple-choice scoring:
+- `aexams_{subject_english}`
```

Comment:
I think it would be helpful to list all `IslamicStudies`, `Biology`, `Science`, `Physics`, `Social` to remove ambiguity for anyone who is unfamiliar.